api/dockerfile

```
@@ -0,0 +1,16 @@
+from node:alpine
```

All instructions should be capitalised by docker convention. https://docs.docker.com/engine/reference/builder/#:~:text=The%20instruction%20is%20not%20case,in%20a%20Dockerfile%20in%20order.
```suggestion
FROM node:alpine
```